src/common/src/platform_posix_exception.cpp

```diff
@@ -70,8 +70,12 @@ dispatchException(Exception *exception,
          return;
       } else {
          // Exception handled, switch execution to target function
-         auto ctx = reinterpret_cast<ucontext *>(context);
+         auto ctx = reinterpret_cast<ucontext_t *>(context);
+#ifdef __APPLE__
```

Comment:
Instead of using __APPLE__ in code directly, can you use PLATFORM_APPLE?

https://github.com/decaf-emu/decaf-emu/blob/master/src/common/platform.h#L6